SimpleCom/SimpleCom.cpp

```diff
@@ -175,7 +178,7 @@ int _tmain(int argc, LPCTSTR argv[])
 	SetCommState(hSerial, &dcb);
 	PurgeComm(hSerial, PURGE_TXABORT | PURGE_RXABORT | PURGE_TXCLEAR | PURGE_RXCLEAR);
 	SetCommMask(hSerial, EV_RXCHAR);
-	SetupComm(hSerial, 1, 1);
+	SetupComm(hSerial, READ_BUF_SIZE, 256);
```

Comment:
I understood we should set `READ_BUF_SIZE` to `dwInQueue`, however I'm not sure we can set `256` to `dwOutQueue` because StdInRedirector would write chars immediately if the user type any keys - so I named it to "Redirector".

I concern that keystrokes will not be reflected to peripheral immediately. Should we flush the handle in each write events? or are there any functions to disable internal buffer?

Reply:
From [the documentation](https://docs.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-setupcomm#remarks), the `SetupCom` parameters seem to be recommendations to the driver. Setting the send driver buffer to greater than `1` doesn't prevent flushing keystrokes.

Reply:
Ok, then could you define "256" as the constant? A simple name is fine.